### ISSUE TITLE
CI: travis-36-cov builds regularly fails to build with The build has been terminated on 1.1.x

### ISSUE THREAD

**Issue by simonjayhawkins** [pandas-dev/pandas#37309]
The last good build can be seen at https://travis-ci.org/github/pandas-dev/pandas/builds/736681105

There are some failures that have started to appear since this started timing out that also need to be resolved.

```
=========================== short test summary info ============================
FAILED pandas/tests/test_downstream.py::test_pandas_gbq - AttributeError: mod...
FAILED pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_to_sql_with_negative_npinf[load_iris_data0-input0]
FAILED pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_to_sql_with_negative_npinf[load_iris_data0-input1]
FAILED pandas/tests/io/test_sql.py::TestMySQLAlchemy::test_to_sql_with_negative_npinf[load_iris_data0-input2]
FAILED pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_to_sql_with_negative_npinf[load_iris_data0-input0]
FAILED pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_to_sql_with_negative_npinf[load_iris_data0-input1]
FAILED pandas/tests/io/test_sql.py::TestMySQLAlchemyConn::test_to_sql_with_negative_npinf[load_iris_data0-input2]
FAILED pandas/tests/util/test_show_versions.py::test_show_versions - Attribut...
ERROR pandas/tests/io/test_gbq.py - AttributeError: module 'google.protobuf.d...
= 8 failed, 73783 passed, 1033 skipped, 1053 xfailed, 6 xpassed, 77 warnings, 1 error in 1533.46s (0:25:33) =
```

xref #36779, https://github.com/pandas-dev/pandas/pull/37303#issuecomment-713513751, https://github.com/pandas-dev/pandas/pull/37304#issuecomment-713515608

**Comment by simonjayhawkins** [comment]
xref #36436 for gbq failures

**Comment by simonjayhawkins** [comment]
xref #36465 for pymysql failures

**Comment by simonjayhawkins** [comment]
closed by #37311
